### PR TITLE
Bump charlock_holmes to 0.7.5

### PR DESCRIPTION
**Pull request by nobuti**
Recently I installed Carto on a new brand laptop, and when I went to `bundle install` I had some issues with charlock_holmes gem:

```
Building native extensions.  This could take a while...
ERROR:  Error installing charlock_holmes:
    ERROR: Failed to build gem native extension.
    current directory: /Users/buti/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/charlock_holmes-0.7.2/ext/charlock_holmes
/Users/buti/.rbenv/versions/2.2.3/bin/ruby -r ./siteconf20170825-25745-g0t4kc.rb extconf.rb
checking for main() in -licui18n... no
checking for main() in -licui18n... yes
checking for unicode/ucnv.h... yes
checking for main() in -lz... yes
checking for main() in -licuuc... yes
checking for main() in -licudata... yes
creating Makefile
current directory: /Users/buti/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/charlock_holmes-0.7.2/ext/charlock_holmes
make "DESTDIR=" clean
current directory: /Users/buti/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/charlock_holmes-0.7.2/ext/charlock_holmes
make "DESTDIR="
compiling converter.c
In file included from converter.c:2:
./common.h:23:14: warning: unused function 'charlock_new_str' [-Wunused-function]
static VALUE charlock_new_str(const char *str, size_t len)
             ^
./common.h:32:14: warning: unused function 'charlock_new_str2' [-Wunused-function]
static VALUE charlock_new_str2(const char *str)
             ^
2 warnings generated.
compiling encoding_detector.c
In file included from encoding_detector.c:2:
./common.h:14:14: warning: unused function 'charlock_new_enc_str' [-Wunused-function]
static VALUE charlock_new_enc_str(const char *str, size_t len, void *encoding)
             ^
1 warning generated.
compiling ext.c
In file included from ext.c:1:
./common.h:14:14: warning: unused function 'charlock_new_enc_str' [-Wunused-function]
static VALUE charlock_new_enc_str(const char *str, size_t len, void *encoding)
             ^
./common.h:23:14: warning: unused function 'charlock_new_str' [-Wunused-function]
static VALUE charlock_new_str(const char *str, size_t len)
             ^
./common.h:32:14: warning: unused function 'charlock_new_str2' [-Wunused-function]
static VALUE charlock_new_str2(const char *str)
             ^
3 warnings generated.
compiling transliterator.cpp
In file included from transliterator.cpp:5:
In file included from /usr/local/Cellar/icu4c/59.1/include/unicode/translit.h:25:
In file included from /usr/local/Cellar/icu4c/59.1/include/unicode/unistr.h:33:
/usr/local/Cellar/icu4c/59.1/include/unicode/char16ptr.h:90:19: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
    Char16Ptr() = delete;
                  ^
/usr/local/Cellar/icu4c/59.1/include/unicode/char16ptr.h:198:24: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
    ConstChar16Ptr() = delete;
                       ^
In file included from transliterator.cpp:5:
In file included from /usr/local/Cellar/icu4c/59.1/include/unicode/translit.h:25:
/usr/local/Cellar/icu4c/59.1/include/unicode/unistr.h:3025:7: error: delegating constructors are permitted only in C++11
      UnicodeString(ConstChar16Ptr(text)) {}
      ^~~~~~~~~~~~~
/usr/local/Cellar/icu4c/59.1/include/unicode/unistr.h:3087:7: error: delegating constructors are permitted only in C++11
      UnicodeString(ConstChar16Ptr(text), length) {}
      ^~~~~~~~~~~~~
/usr/local/Cellar/icu4c/59.1/include/unicode/unistr.h:3180:7: error: delegating constructors are permitted only in C++11
      UnicodeString(Char16Ptr(buffer), buffLength, buffCapacity) {}
      ^~~~~~~~~~~~~
2 warnings and 3 errors generated.
make: *** [transliterator.o] Error 1
make failed, exit code 2
Gem files will remain installed in /Users/buti/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/charlock_holmes-0.7.2 for inspection.
Results logged to /Users/buti/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/extensions/x86_64-darwin-16/2.2.0-static/charlock_holmes-0.7.2/gem_make.out
```

For some reason, C++11 was not available to gcc when compiling `icu`. But the thing is, version 0.7.5 already does it as part the gem itself: https://github.com/brianmario/charlock_holmes/pull/116. 

It's a minor bump, so I'm confident it will not break anything. Tested locally and it works without any issue. Time to test it more deeply.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -441,4 +441,4 @@ DEPENDENCIES
   zeus
 
 BUNDLED WITH
-   1.14.6
+   1.15.4
```
Comment (javitonino) [inline Gemfile.lock]: Please, manually revert this line. I know it is autogenerated, but we don't like bumping it.

**Comment by Jesus89** [comment]
I have also found that bug, previously reported to https://github.com/gfiorav/carto_loves_mac/issues/12

**Comment by nobuti** [comment]
It's in ded04, I did some tests and I could not see any issue.

**Comment by nobuti** [comment]
Not in ded04 anymore.

**Comment by xavijam** [comment]
So? could we merge it?

**Comment by nobuti** [comment]
I couldn't see any issue but I'd sleep better if someone else could take a look.

**Comment by javitonino** [comment]
Necrobumping this one. I'll do some quick acceptance and merge if everything is OK.

**Pull request merged**